src/main/java/io/jenkins/plugins/coverage/adapter/CoberturaReportAdapter.java

```diff
@@ -35,6 +36,7 @@ public String getXSD() {
         return null;
     }
 
+    @Symbol("cobertura")
```

Comment:
I wonder if we need a new symbol here, so it doesn't conflict with the cobertura plugin? @oleg-nenashev - what do you think?

Reply:
Ah - I see this change was suggested by @oleg-nenashev in a previous review, please disregard.

Reply:
Maybe we should add a simple prefix or suffix for this.

Reply:
I was mostly concerned it might somehow interfere with the existing cobertura plugin in the snippet builder, but if that's not a problem, I think it's OK.

Reply:
There should be no conflicts between symbols as long as they declared for different extension points